src/services/user_service.ts

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+
+export function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId)
+  || null;
```

Comment:
`|| null` is redundant cause method find always return an array 
```suggestion
  return usersFromServer.find(user => user.id === userId);
```